file-proxy/encrypted-file-proxy.js

```diff
@@ -0,0 +1,98 @@
+"use latest"
+
+import Webtask from 'webtask-tools';
+import express from 'express';
+import crypto from 'crypto';
+import multiparty from 'multiparty';
+import formData from 'form-data';
+import request from 'request';
+
+const app = express();
+
+// The upload endpoint
+app.post('/:projectid', (req, res) => {
```

Comment:
Something is definitely off with that change tracking when you make changes. It's easier to commit them to the PR like @marktani did above. Anyways, the projectId was one of the things I overlooked having to do a manual merge of your previous changes... I will update that. What other things did you change? Only the use of graphql-request in encrypted-file-proxy?